OmniBLE/PumpManagerUI/ViewModels/OmniBLESettingsViewModel.swift

```diff
@@ -130,8 +140,12 @@ class OmniBLESettingsViewModel: ObservableObject {
             return LocalizedString("Insulin delivery stopped. Change Pod now.", comment: "The action string on pod status page when pod faulted")
         } else if podOk && isPodDataStale {
             return LocalizedString("Make sure your phone and pod are close to each other. If communication issues persist, move to a new area.", comment: "The action string on pod status page when pod data is stale")
-        } else if let podTimeRemaining = pumpManager.podTimeRemaining, podTimeRemaining < 0 {
-            return LocalizedString("Change Pod now. Insulin delivery will stop 8 hours after the Pod has expired or when no more insulin remains.", comment: "The action string on pod status page when pod expired")
+        } else if let serviceTimeRemaining = pumpManager.podServiceTimeRemaining, serviceTimeRemaining <= Pod.serviceDuration - Pod.nominalPodLife {
+            if let serviceTimeRemainingString = serviceTimeRemainingString {
+                return String(format: LocalizedString("Change Pod now. Insulin delivery will stop in %@ or when no more insulin remains.", comment: "Format string for the action string on pod status page when pod expired"), serviceTimeRemainingString)
```

Comment:
Adding a placeholder comment is nice for translators to know what the positional argument will be substituted with. Something like:
```
String(format: NSLocalizedString("%1$@ v%2$@", comment: "The format string for the app name and version number. (1: bundle name)(2: bundle version)"), bundleDisplayName, fullVersionString)
```

Reply:
Will do! Thanks!